code/game/objects/effects/landmarks.dm

```diff
@@ -1,85 +1,44 @@
+var/list/landmarks_list = list() //list of all landmarks
+var/list/landmarks_by_type = list() //list of landmark types associated with turf lists
+
+//Returns a list of all landmark turfs or an empty list
+/proc/get_landmarks(input_type, readonly = TRUE)
+	var/list/L = landmarks_by_type[input_type]
+	return L ? (readonly ? L : L.Copy()) : list()
```

Comment:
This is some relatively ugly nesting. I might do `L = landmarks_by_type[input_type] || list()` to simplify it. Otherwise, there's nothing wrong with being verbose.